option.go

```diff
@@ -118,13 +119,15 @@ func (o *Option) Doc() *Comment {
 	return o.Comment
 }
 
-// Literal represents intLit,floatLit,strLit or boolLit
+// Literal represents intLit,floatLit,strLit or boolLit or a nested structure thereof.
 type Literal struct {
 	Position scanner.Position
 	Source   string
```

Comment:
I don't see how this is set or not with nested structures - documentation + test?